web/src/locales/pt.json

```
@@ -76,5 +76,13 @@
 	"notFound.description": "A página que procura não existe.",
 	"forbidden.title": "Acesso proibido",
 	"forbidden.description": "Não tem autorização para aceder a esta página.",
-	"map.layers": "Camadas"
+	"map.layers": "Camadas",
+	"employees.title": "Colaboradores",
+	"employees.search": "Pesquise por nome de utilizador",
+	"employees.username": "Nome de utilizador",
+	"employees.name": "Nome",
+	"employees.schedule": "Horário",
+	"employees.role": "Categoria",
```

What do you think about renaming this to `Função`?
```suggestion
	"employees.role": "Função",
```